rules/falco_rules.yaml

```diff
@@ -2676,7 +2676,7 @@
   tags: [file, mitre_persistence]
 
 - list: remote_file_copy_binaries
-  items: [rsync, scp, sftp, dcp]
+  items: [rsync, scp, sftp, dcp, wget, curl]
```

Comment:
actually I think `curl` might cause a lot of false positive here. As the `curl` is used as a readiness probe or liveness probe commonly.

Reply:
Likely true, but it's also commonly to curl and redirect `>` the output to a file. What do you think we should do? 😄

Reply:
After thinking a bit, probably we should differentiate the case of `copy in` and `copy out`, tools like `rsysnc`, `scp` and `scp` more likely fall into the bucket of `copy out` from the container. While `wget` and `curl` fall into the `copy in` bucket. Do you mind to create another rule for it ?

Reply:
I'll look into it :)

Reply:
@Kaizhe please take another look. I moved wget and curl into its own "ingress" rule https://github.com/falcosecurity/falco/pull/1771/commits/05d5d09251dc713f143536ee7a1208237de47580

Reply:
@Kaizhe bump :)